app/src/main/kotlin/com/intive/picover/profile/model/ProfileActionsListItem.kt

```diff
@@ -0,0 +1,31 @@
+package com.intive.picover.profile.model
+
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import com.intive.picover.R
+
+sealed class ProfileActionsListItem(val onClick: () -> Unit, @StringRes val textId: Int, @DrawableRes val iconId: Int) {
```

Comment:
`ProfileActionItem` would be enough, there is no need decorate the name too much

Reply:
Done

Reply:
Minor: use singular form "Action", not "Actions"